packages/veraswap-sdk/src/uniswap/getRouteMultichain.ts

```diff
@@ -60,6 +61,16 @@ export async function getRouteMultichain(
         return { flows: [{ type: "BRIDGE", currencyIn, currencyOut }], amountOut: exactAmount };
     }
 
+    if (
+        currencyIn.symbol &&
+        currencyOut.symbol &&
+        currencyIn.symbol === currencyOut.symbol &&
+        currencyIn.symbol in STARGATE_TOKEN_POOLS &&
+        currencyOut.symbol in STARGATE_TOKEN_POOLS
```

Comment:
Remove last check. Redundant since `currencyIn.symbol === currencyOut.symbol` and `currencyIn.symbol in STARGATE_TOKEN_POOLS`